lib/client/connection.js

```diff
@@ -454,11 +454,13 @@ Connection.prototype.sendOp = function(doc, op) {
     d: doc.id,
     v: doc.version,
     src: op.src,
-    seq: op.seq
+    seq: op.seq,
+    x: {}
   };
   if ('op' in op) message.op = op.op;
   if (op.create) message.create = op.create;
   if (op.del) message.del = op.del;
+  if (doc.submitSource) message.x.source = op.source;
```

Comment:
I wonder if `submitSource` reads clear enough as a flag?

Though I must admit I struggle to think of a better one 🤔 

Maybe `recordSource` or `trackOpSource`

Reply:
Yeah I mean it kind of sounds like a method I guess. Could be `shouldSubmitSource`/`sourceIsSubmitted`? But starts getting verbose

Reply:
`keepTrackOfSourceWhenSubmittingOps`? 😄